ITR/data/excel.py

```diff
@@ -102,8 +105,8 @@ def _get_projected_production(self, scope: EScope = EScope.S1S2) -> pd.DataFrame
 
 
 class ExcelProviderIntensityBenchmark(BaseProviderIntensityBenchmark):
-    def __init__(self, excel_path: str, benchmark_temperature: Quantity['delta_degC'],
-                 benchmark_global_budget: Quantity['CO2'], is_AFOLU_included: bool,
+    def __init__(self, excel_path: str, benchmark_temperature: quantity('delta_degC'),
```

Comment:
The `Quantity` class should be powerful enough, to init all it needs. Why do we need a `quantity` wrapper around it?

Which value does it bring in this concrete case?

Reply:
As explained in another Resolved conversation, Quantity is a pint thing, taking as arguments a magnitude and a unit. 
 quantity was written as a Pydantic thing--a semi-generic type that maps to a Quantity, but which can be validated as having a value with the units given.